app/_landing_pages/dev-portal.yaml

```
@@ -151,6 +145,16 @@ rows:
                 Change the Dev Portal UI appearance.
               cta:
                 url: /dev-portal/portal-customization/
+  - column_count: 3
+    columns:
+      - blocks:
+          - type: card
+            config:
+              title: Email customization
+              description: |
+                Learn how to cusomize the content and domain for the emails developers receive for Dev Portal events.
```

```suggestion
                Learn how to customize the content and domain for the emails developers receive for Dev Portal events.
```
Not sure why vale didn't catch this 🤔 is it broken again?